chain/network/src/broadcast/mod.rs

```diff
@@ -47,6 +47,16 @@ impl<T: Send + Sync + 'static> Sender<T> {
 }
 
 impl<T: Clone + Send> Receiver<T> {
+    // Returns a copy of the receiver which
+    // ignores all the events until now.
+    // TODO(gprusak): this still doesn't solve
+    // the events being mixed in the stream.
+    // Without actix, awaiting the expected state
+    // should get way easier.
+    pub fn from_now(&self) -> Self {
```

Comment:
Worthwhile to mark this as `#[cfg(test)]` if only as a “documentation” that this is a test-only function. This would also ensure the function doesn’t accidentally leak as a public API.

Speaking of that… does this need to be a `pub`? Could it be a `pub(crate)`?

Reply:
This whole package is testonly. The package is public, because integration tests are using it currently. IMO it would be misleading to additionally label stuff with `#[cfg(test)]` within a testonly package.